siriuspy/siriuspy/devices/psconv.py

```diff
@@ -103,9 +102,9 @@ def family_strength(self):
             return self._dev_fam.value
         return None
 
-    def conv_current_2_strength(self, currents,
-                                strengths_dipole=None,
-                                strengths_family=None):
+    def conv_current_2_strength(
+            self, currents,
```

Comment:
Is pylint ok with this?

Reply:
it is.
![screenshot_2020-03-30_18h01m02s](https://user-images.githubusercontent.com/5631815/77961620-7e7aa980-72b0-11ea-9964-b31115e84620.png)

Reply:
Good to know!